charts/newrelic-infrastructure-v3/tests/configmap_agent_test.yaml

```diff
@@ -0,0 +1,63 @@
+suite: test configmap
+templates:
+  - templates/agent-configmap.yaml
+tests:
```

Comment:
I like this unit tests :)

Reply:
what do you think to move the compatibility related test to an specific folder?

Reply:
done

Reply:
actually I noticed that tests are ignored if placed in a separate folder 😖